src/.whitelist

```diff
@@ -0,0 +1,3 @@
+[DEFAULT]
+ip1 = 127.0.0.1
+ip2 = 192.168.182.161
```

Comment:
Nice!

Should we support subnets in future (e.g. 192.168.0.0/24)? No rush though.